shared/actions/chat/index.js

```diff
@@ -242,118 +242,120 @@ function * _ensureValidSelectedChat (onlyIfNoSelection: boolean, forceSelectOnMo
 function * _loadMoreMessages (action: Constants.LoadMoreMessages): SagaGenerator<any, any> {
   const conversationIDKey = action.payload.conversationIDKey
 
-  if (!conversationIDKey) {
```

Comment:
This is me wrapping this in a try / finally block as this thing can be cancelled. In this case it'll always set the loading state to false. This wasn't what was happening here but is technically possible. use https://github.com/keybase/client/pull/6917?w=1 for a cleaner diff